SpatialGDK/Source/SpatialGDK/Private/SpatialView/WorkerView.cpp

```diff
@@ -90,30 +90,38 @@ TUniquePtr<MessagesToSend> WorkerView::FlushLocalChanges()
 
 void WorkerView::SendAddComponent(Worker_EntityId EntityId, ComponentData Data)
 {
-	EntityViewElement& Element = View.FindChecked(EntityId);
-	Element.Components.Emplace(Data.DeepCopy());
-	LocalChanges->ComponentMessages.Emplace(EntityId, MoveTemp(Data));
+	EntityViewElement* Element = View.Find(EntityId);
+	if (ensure(Element != nullptr))
```

Comment:
Should these be ensureAlways? Although you could after the first one you have to assume that something is broken and other errors are suspicious.

I guess with ensure there isn't really a nice way to reduce nesting by returning in the failure branch.